bazel/repository_locations.bzl

```diff
@@ -790,21 +790,50 @@ REPOSITORY_LOCATIONS_SPEC = dict(
-        # and contains complete checkout of V8 with all dependencies necessary to build wee8.
-        sha256 = "4f4353928d10adbc07503edcb7dbd5a20981de669225dcffc450472cbfb179c2",
-        urls = ["https://storage.googleapis.com/envoyproxy-wee8/wee8-{version}.tar.gz"],
-        strip_prefix = "wee8",
+        # NOTE: Update together with com_googlesource_chromium_base_trace_event_common and com_googlesource_chromium_zlib.
+        version = "9.9.115.5",
+        # Static snapshot created using https://storage.googleapis.com/envoyproxy-wee8/wee8-fetch-deps.sh.
+        sha256 = "60a6759069350571b24b65eb88c2c1908afed2b938dae6a92b305924f3b64e84",
+        urls = ["https://storage.googleapis.com/envoyproxy-wee8/v8-{version}.tar.gz"],
         use_category = ["dataplane_ext"],
         extensions = ["envoy.wasm.runtime.v8"],
-        release_date = "2022-01-24",
+        release_date = "2022-01-31",
         cpe = "cpe:2.3:a:google:v8:*",
     ),
+    com_googlesource_chromium_base_trace_event_common = dict(
+        project_name = "Chromium's trace event headers",
+        project_desc = "Chromium's trace event headers",
+        project_url = "https://chromium.googlesource.com/chromium/src/base/trace_event/common/",
+        # NOTE: Update together with v8 and com_googlesource_chromium_zlib.
+        # Use version and sha256 from https://storage.googleapis.com/envoyproxy-wee8/v8-<v8_version>-deps.sha256.
+        version = "7f36dbc19d31e2aad895c60261ca8f726442bfbb",
+        # Static snapshot created using https://storage.googleapis.com/envoyproxy-wee8/wee8-fetch-deps.sh.
+        sha256 = "4e9f31f393ea87bf3987f9a5e3b1d8314ad45f00e5a79c61cdcc1ef1eed2f05b",
+        urls = ["https://storage.googleapis.com/envoyproxy-wee8/chromium-base_trace_event_common-{version}.tar.gz"],
+        use_category = ["dataplane_ext"],
+        extensions = ["envoy.wasm.runtime.v8"],
+        release_date = "2021-10-27",
+        cpe = "N/A",
+    ),
+    com_googlesource_chromium_zlib = dict(
+        project_name = "Chromium's zlib",
+        project_desc = "Chromium’s fork of zlib with compression utils",
+        project_url = "https://chromium.googlesource.com/chromium/src/third_party/zlib/",
+        # NOTE: Update together with v8 and com_googlesource_chromium_base_trace_event_common.
+        # Use version and sha256 from https://storage.googleapis.com/envoyproxy-wee8/v8-<v8_version>-deps.sha256.
+        version = "fc5cfd78a357d5bb7735a58f383634faaafe706a",
+        # Static snapshot created using https://storage.googleapis.com/envoyproxy-wee8/wee8-fetch-deps.sh.
+        sha256 = "695c73750cf6472fc6c926e43952262206f1475157377364142bdbb84a1a5a83",
+        urls = ["https://storage.googleapis.com/envoyproxy-wee8/chromium-zlib-{version}.tar.gz"],
+        use_category = ["dataplane_ext"],
+        extensions = ["envoy.wasm.runtime.v8"],
+        release_date = "2022-01-12",
+        cpe = "N/A",
+    ),
```

Comment:
Any concerns with this being the 3rd zlib dependency we bring into Envoy /cc @envoyproxy/dependency-shepherds

Reply:
Note that we're not using this as a zlib library, we're linking against whatever `//external:zlib` points at.

Unfortuantely, V8 uses tiny C++ shim library that's included in their fork (the "zlib compression utils"), which we have to use, but similarly, that's linked against `//external:zlib` in this build.

Lastly, this is still an improvement over the existing `genrule_repository()` that we use for V8, which links against Chromium's zlib fork.